src/privateJrPlatform/getJrPlatformApp.ts

```diff
@@ -0,0 +1,40 @@
+import { load, Obj } from 'scrivito'
+import { App } from '../App'
+import { WrongContentFormat } from './Components/WrongContentFormat'
+
+export async function getJrPlatformApp() {
+  return (await isValidContentFormat()) ? App : WrongContentFormat
+}
+
+const CONTENT_FORMAT = 'portal-app:5'
+const KNOWN_CONTENT_FORMATS: Record<string, string> = {
+  'portal-app:5': 'https://scrivito-portal-app.pages.dev',
+
+  /* Once v6 is released, the list of KNOWN_CONTENT_FORMATS looks like this: */
+  // 'portal-app:5': 'https://v5.scrivito-portal-app.pages.dev',
+  // 'portal-app:6': 'https://scrivito-portal-app.pages.dev',
+}
+
+/** Redirects away, if content format is not valid. */
+async function isValidContentFormat(): Promise<boolean> {
+  const root = await load(() => Obj.root())
+  if (!root) return true
+
+  const siteContentFormat = root.get('contentFormat')
+  if (siteContentFormat === CONTENT_FORMAT) return true
+
+  if (typeof siteContentFormat !== 'string') return false
+  if (!siteContentFormat) return false
+
+  const redirectOrigin = KNOWN_CONTENT_FORMATS[siteContentFormat]
+  if (!redirectOrigin) return false
+
+  const url = location.href.replace(location.origin, redirectOrigin)
```

Comment:
* [ ] FYI in local testing this can lead to an endless redirect loop. It would be great if this case (redirect to same origin?) would show the `WrongContentFormat` instead. What I did:


* Set the content to v6
* Added a v6 redirect to http://v6.localhost:8080
* The redirect from http://localhost:8080 worked as expected but
* Since I left the app on v5, which was required for the test, there was an endless loop